### PR TITLE
added link to an ssh guide

### PR DESCRIPTION
**Pull request by panahiparham**
I added this link because it contains a short tutorial introduction to ssh and also some information on scp

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,7 @@
 ## Category 1
 
 * [Placeholder link]()
+
+## How Do I connect to the Cluster?
```
Comment (andy-amii) [inline README.md]: Hmm, I suppose we should decide whether we want these to be title-case or complete sentences. I could go either way.

I think I'm leaning towards complete sentences for now, so something like:
```markdown
## How do I connect to the cluster?
```

Any opinions or preferences?

Comment (andy-amii) [reply README.md]: We can also go ahead and create a table-of-contents entry for this. I _believe_ for github-flavored markdown, this looks like:
```markdown
2. [How do I connect to the cluster?](#how-do-i-connect-to-the-cluster)
```

File: README.md [hunk 2]
```diff
@@ -11,3 +11,7 @@
...
+
+* [SSH quickstart](https://linuxhandbook.com/ssh-basics/)
```
Comment (andy-amii) [inline README.md]: I like the idea of adding short descriptions to the links, separated by a hyphen. Something like:
```markdown
* [SSH quickstart](link) - Step-by-step ssh and scp guide.
```

For the descriptions, I think we should target keywords that make the list more searchable.

**Review by andy-amii — CHANGES_REQUESTED** [review]
Sorry your PR is getting the bulk of the "request changes" while we figure out the format!

File: README.md (hunk 1 above)
Comment (panahiparham) [reply README.md]: Complete sentences are better

File: README.md (hunk 2 above)
Comment (panahiparham) [reply README.md]: sounds good

**Comment by panahiparham** [comment]
Updated requested formatting changes

**Review by andy-amii — APPROVED** [review]

**Pull request merged**